extensions/amp-video/0.1/test-e2e/test-amp-video-analytics.js

```diff
@@ -46,9 +46,9 @@ describes.endtoend(
       await expect(isPaused(videoElem1)).to.be.false;
       await expect(isPaused(videoElem2)).to.be.true;
 
-      // Sleep 5 seconds for the `video-percentage-played` event trigger
+      // Sleep 1 second for the `video-percentage-played` event trigger
```

Comment:
@micajuine-ho is it necessary that these be synchronous? Or could we play both videos at the same time and get the same behavior with only a single call to `sleep`?

Reply:
I think it's best if they're sync, RequestBank (that captures requests) has been finicky in the past so keeping it sync should help prevent potential future flakes.

Reply:
Alright, that makes sense but it seemed worth checking.